test/e2e/utils/kubernetes.go

```diff
@@ -155,20 +156,30 @@ func (k TestKubernetes) CleanNamespaceAndLogOnPanic(t *testing.T, namespace stri
 }
 
 func (k TestKubernetes) CleanNamespaceAndLogWithPanic(t *testing.T, namespace string, panicVal interface{}) {
-	testName := TestName(t)
-	specLabel := map[string]string{"test-name": testName}
-
-	// Print pod output if a panic has occurred
-	if panicVal != nil {
-		k.PrintAllResources(namespace, &corev1.PodList{}, map[string]string{"app.kubernetes.io/name": "infinispan-operator"})
-		k.PrintAllResources(namespace, &corev1.PodList{}, map[string]string{"app": "infinispan-pod"})
-		k.PrintAllResources(namespace, &corev1.PodList{}, map[string]string{"app": "infinispan-batch-pod"})
-		k.PrintAllResources(namespace, &appsv1.StatefulSetList{}, map[string]string{})
-		k.PrintAllResources(namespace, &ispnv1.InfinispanList{}, map[string]string{})
-		k.PrintAllResources(namespace, &ispnv2.BackupList{}, map[string]string{})
-		k.PrintAllResources(namespace, &ispnv2.RestoreList{}, map[string]string{})
-		k.PrintAllResources(namespace, &ispnv2.BatchList{}, map[string]string{})
-		k.PrintAllResources(namespace, &ispnv2.CacheList{}, map[string]string{})
+	specLabel := make(map[string]string)
+	if t != nil {
```

Comment:
I wonder in what use cases the `t` is nil?

Reply:
There's not a specific case in the `2.2.x` code base atm where we pass a nil `t`, but I figured backporting the check now has no consequences and will make subsequent backports related to this code simpler as this check is need on `main`.